### PR TITLE
Brandons/dev

### PR DESCRIPTION
**Pull request by Bricktheworld**
Most of everything should work in this. I think the README.md is up to date, but just check it to be sure, same thing with package.json. Also install process has not been verified yet but I will once this gets merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Bricktheworld/RhombusIdentifier.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Bricktheworld/RhombusIdentifier/issues"
```
Comment (rhombusdev) [inline package.json]: change to the RhombusSystems github

File: src/services/classifier.ts [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+		boundingBoxes.push({
+			a: ActivityEnum.CUSTOM,
+			// These values are permyriads, so we need to convert our bounding boxes appropriately
+			b: (box.position.y + box.dimensions.y) / dimensions.y * 10000, // Bottom
+			l: (box.position.x / dimensions.x) * 10000, // Left
+			r: ((box.position.x + box.dimensions.x) / dimensions.x) * 10000, // Right
+			t: (box.position.y / dimensions.y) * 10000, // Top
+			ts: box.timestamp,
+			cdn: box.label,
+			ro: [box.label],
```
Comment (rhombusdev) [inline src/services/classifier.ts]: this field `ro` is technically optional and is not used in the Rhombus console at this moment.

**Review by rhombusdev — COMMENTED** [review]
Nice work.  I would move all of the required files to a directory `ExtendedAIModule/` or something similar to keep this example self contained if we add others in the future.  The README would have to be adjusted to point to the directory.

**Comment by Bricktheworld** [comment]
Ok I have moved everything into the ExtendedAIModule directory and removed the recognized objects parameter. I also updated the README to make things more flexible when we add more modules

**Pull request merged**